### ISSUE TITLE
how to resolve the catastrophic forgetting about personalized fine-tuning on sd

### ISSUE THREAD

**Issue by zhouyifeng888** [mindspore-lab/minddiffusion#57]
May I ask if the personalized fine-tuning method of the Wukong model can be applied to stable diffusion? I tried it, and it could run, but there was catastrophic forgetting in stable diffusion. How can I solve this problem? If the learning rate is set exactly the same as the Wukong model, it seems that it cannot learn the features of new images, and the fine-tuning effect does not work, but catastrophic forgetting still occurs. If the learning rate is increased, such as 10 times the personalized fine-tuning of the Wukong model, it can learn the features of new images very well, but catastrophic forgetting becomes more severe.

**Comment by congw729** [comment]
1. Try using different learning rates or other hyperparameters;
2. Make sure your current training dataset is not trained before;
